docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+go:
+  build: .
+  links: 
+    - mysql
+  command: make run-server
+
+mysql:
+  build: images/mysql
+  ports:
+    - "3307:3306"
+  environment:
+    - MYSQL_ROOT_PASSWORD=root
+    - MYSQL_DATABASE=jwt_api
+    - MYSQL_USER=test_user
+    - MYSQL_PASSWORD=Qwerty1
+
+nginx:
```

Comment:
In this example nginx is a bit overhead, you don't have scaling here or ant request "restrictions". If you want - you can keep this, if not - remove